### PR TITLE
EPMLSTRCMW-124: Move implicit formatters

### PR DESCRIPTION
**Pull request by AndreiMaksimenko**
Custom exception handlers are already done with User micro types.
Implicit formatters for User, Project, and Auth was moved
to formatters package.
Micro types implicit moved as well.

### CODE REVIEW

**Review by GrigoriiBerezin — APPROVED** [review]
Nothing to add. Good job

**Review by IaroslavTavchenkov — APPROVED** [review]
Well-done!

**Review by OstapStrashevskii — APPROVED** [review]
Nice work, all format implicits ordered in special places!

**Review by iliaonishchenko — APPROVED** [review]

**Review by myazinn — CHANGES_REQUESTED** [review]
What about formatters for our microtypes? I'm' talking about both the generic method `wrapperFormat` and his implementations like `userEmailFormat`.

File: repositories/src/main/scala/cromwell/pipeline/datastorage/formatters/UserFormatters.scala [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  implicit lazy val profilePictureFormat: OFormat[ProfilePicture] = Json.format[ProfilePicture]
+  implicit val UserNoCredentialsFormat: OFormat[UserNoCredentials] =
+    Json.format[UserNoCredentials]
+  implicit lazy val updatePasswordRequestFormat: OFormat[PasswordUpdateRequest] = Json.format[PasswordUpdateRequest]
+  implicit lazy val updateRequestFormat: OFormat[UserUpdateRequest] = Json.format[UserUpdateRequest]
+
+  implicit lazy val nameFormat: Format[Name] = Name.wrapperFormat
+  implicit lazy val passwordFormat: Format[Password] = Password.wrapperFormat
+  implicit lazy val userEmailFormat: Format[UserEmail] = UserEmail.wrapperFormat
+  implicit lazy val userIdFormat: Format[UserId] = UserId.wrapperFormat
```
Comment (myazinn) [inline repositories/src/main/scala/cromwell/pipeline/datastorage/formatters/UserFormatters.scala:19]: This is cheating :)
`wrapperFormat` must also be moved from `Wrapped` object into something like `WrappedFormatters` (maybe this is a bad name and there is a better one). You can put all formatters for these wrappers into that object.

**Review by myazinn — CHANGES_REQUESTED** [review]

**Comment by iliaonishchenko** [comment]
I'm closing this MR because @OstapStrashevskii is working on this right now. 
And work is done in https://github.com/EpamLifeSciencesTeam/cmwl_pipeline/pull/90

**Pull request closed without merging**